ores/lock_manager/poolcounter.py

```diff
@@ -81,6 +88,7 @@ def close(self):
     @classmethod
     def from_config(cls, config, name, section_key="lock_managers"):
         nodes = []
+        # TODO: Fix config so we can inject connection_timeout
```

Comment:
Yes please. It would be great for this to be configurable

Reply:
Yes, let's fix this issue at hand first. I make a phabricator ticket.